Keypad/Keypad.js

```diff
@@ -161,18 +162,28 @@ const KeypadBase = kind({
 		return (
 			<Layout {...rest} align="center end" className={css.keypad} inline wrap>
 				{KEY_LIST.map((keyText, rowIndex) => {
+					const {icon, text} = keyText;
+					const isIcon = icon === 'arrowleftturn' || icon === 'phone';
+
+					let ariaLabel = text;
+					if (icon === 'arrowleftturn') {
+						ariaLabel = $L('backspace');
+					} else if (icon === 'phone') {
+						ariaLabel = $L('call');
+					}
```

Comment:
I think that we could simplify the code above more.

```
const KEY_LIST = [
	...
	{icon: 'phone', label: 'call'},
	{icon: 'arrowleftturn', label: 'backspace'}
];
```

```suggestion
const {icon, label, text} = keyText;
const ariaLabel = icon ? $L(label) : text;
```